recipes/tng/all/test_package/example.c

```diff
@@ -0,0 +1,152 @@
+/* This code is part of the tng binary trajectory format.
+ *
+ * Written by Magnus Lundborg
+ * Copyright (c) 2012-2013, The GROMACS development team.
+ * check out http://www.gromacs.org for more information.
+ *
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the Revised BSD License.
+ */
+
+#include "tng/tng_io.h"
+
+#ifdef USE_STD_INTTYPES_H
+#include <inttypes.h>
+#endif
+
+#include <stdlib.h>
+#include <stdio.h>
+
+int main(int argc, char **argv)
```

Comment:
we like to have much move minimal test_packages... I idea it to make sure the conan packing went well no so much the library is functional.

could you please try to reduce this to a few lines? example instantiate a few objects?

Reply:
I used an example piece of code from the library to create the test_package.c, figured it would be a good way to test it.

I will reduce it.